src/sentry/integrations/msteams/integration.py

```diff
@@ -96,6 +96,7 @@ def build_integration(self, state):
                 "expires_at": token_data["expires_at"],
                 "service_url": service_url,
             },
+            # TODO: Use user id for external_id in user_identity
```

Comment:
I feel like this TODO is def going to get lost, do we want to make this into a tech-debt jira card?

Reply:
@MeredithAnya yea, I can make a card